packages/office-ui-fabric-react/src/components/Facepile/Facepile.styles.ts

```diff
@@ -1,21 +1,154 @@
 import { IFacepileStyleProps, IFacepileStyles } from './Facepile.types';
 
-export const getStyles = (
+import {
+  IStyle,
+  hiddenContentStyle,
+  getFocusStyle,
+  getGlobalClassNames,
+} from '../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-Facepile',
+  addButton: 'ms-Facepile-addButton ms-Facepile-itemButton',
+  descriptiveOverflowButton: 'ms-Facepile-descriptiveOverflowButton ms-Facepile-itemButton',
+  itemButton: 'ms-Facepile-itemButton ms-Facepile-person',
+  itemContainer: 'ms-Facepile-itemContainer',
+  members: 'ms-Facepile-members',
+  overflowButton: 'ms-Facepile-overflowButton ms-Facepile-itemButton',
+};
+
+export const styles = (
   props: IFacepileStyleProps
 ): IFacepileStyles => {
   const {
-    className
+    className,
+    theme,
+    spacingAroundItemButton = 2,
   } = props;
 
+  const { palette, fonts } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const ItemButtonStyles: IStyle = {
+    textAlign: 'center',
+    padding: 0,
+    borderRadius: '50%',
+    verticalAlign: 'top',
+    selectors: {
+      'button&': {
```

Comment:
The css in this is really chunky because we're not abstracting the concept. This note isn't really asking you to change this in the PR, but this right here perfectly illustrates how without more atomic abstractions, we end up duplicating concepts, styling, and bloating the code.

Reply:
Agreed - should we track this as a work item somehow for these cases?